### ISSUE TITLE
scss processing

### ISSUE THREAD

**Issue by maurercw** [webjars/datatables-buttons#1]
Can you run a project build so that the scss files can be processed into css

Chris

**Comment by maurercw** [comment]
Any chance we can get some movement on this?

**Comment by jamesward** [comment]
I'm having issues running their build:
```
$ sh make.sh 
make.sh: line 13: cd: /Volumes/Home/j.ward/projects/webjars/datatables-buttons/target/Buttons-1.3.1/../../build: No such file or directory
make.sh: line 14: ./make.sh: Permission denied
make.sh: line 15: cd: OLDPWD not set
```

It seems their build makes some assumptions about where some other stuff is.  Maybe their is another way to do this build?

**Comment by jamesward** [comment]
Can we just use the release from: https://datatables.net/download/packages

**Comment by maurercw** [comment]
Sure, I don't see why not.

**Comment by jamesward** [comment]
Cool.  This will be released shortly as `1.3.1-1`.

**Comment by maurercw** [comment]
Everything looks great, thanks!
